script.js

```diff
@@ -12,3 +12,344 @@ document.addEventListener('DOMContentLoaded', () => {
+          </ul>
+      </div>
+      <p class="mobile">A daily selection of privately personalized reads; no accounts or sign-ups
+          required.</p>
+      <p class="desktop">Exploring the future of media in Facebook's first Virtual Reality app; a place to
+          discover and enjoy 360 photos and videos on Gear VR.</p>
+      <div class="tags">
+          <ul>
+              <li><a>${projects[2].technologies[0]}</a></li>
+              <li class="tag desktop"><a>${projects[2].technologies[1]}</a></li>
+              <li><a>${projects[2].technologies[2]}</a></li>
+              <li><a>${projects[2].technologies[3]}</a></li>
+          </ul>
+      </div>
+      <div data-modal-target="#modal" id="project3" class="seebutton2">
+          <button type="button">See Project</button>
+      </div>
+  </div>
+</div>
+
+
+<div class="card card_4">
+<img class="Snapshoot_Portfolio_4 mobile" src="./images/Snapshot4.png" alt="Snapshoot_Portfolio_4">
+<img class="Snapshoot_Portfolio_4 desktop" src=${projects[3].image} alt="Snapshoot_Portfolio_8">
+<div class="work_cards">
+    <h2 class="mobile">Multi-Post Stories</h2>
+    <h2 class="desktop">${projects[3].title}</h2>
+    <div class="frame mobile">
+        <ul>
+            <li class="canopy">CANOPY</li>
+            <li class="separator"></li>
+            <li>Back End Dev</li>
+            <li class="separator"></li>
+            <li>2015</li>
+        </ul>
+    </div>
+    <div class="frame desktop">
+        <ul>
+            <li class="canopy">Uber</li>
+            <li class="separator"></li>
+            <li>Lead Developer</li>
+            <li class="separator"></li>
+            <li>2018</li>
+        </ul>
+    </div>
+    <p class="mobile">A daily selection of privately personalized reads; no accounts or sign-ups
+        required.</p>
+    <p class="desktop">A smart assistant to make driving more safe, efficient, and fun by unlocking your
+        most expensive computer: your car.</p>
+    <div class="tags">
+        <ul>
+            <li><a>${projects[3].technologies[0]}</a></li>
+            <li class="tags desktop"><a>${projects[3].technologies[1]}</a></li>
+            <li><a>${projects[3].technologies[2]}</a></li>
+            <li><a>${projects[3].technologies[3]}</a></li>
+        </ul>
+    </div>
+    <div data-modal-target="#modal" id="project4" class="seebutton3">
+          <button type="button">See Project</button>
+      </div>
+</div>
+</div>
+`;
+
+// code for apearing and disapearing of modal
+const openButton = document.querySelectorAll('[data-modal-target]');
+const closeButton = document.querySelectorAll('[data-close-button]');
+const backdrop = document.getElementById('backdrop');
+
+function openModal(modal) {
+  if (modal === null) return;
+  modal.classList.add('active');
+  backdrop.classList.add('active');
+}
+
+function closeModal(modal) {
+  if (modal === null) return;
+  modal.classList.remove('active');
+  backdrop.classList.remove('active');
+}
+
```

Comment:
- [ ] Actually, your popup is too large and I'm not able to see the close button and the popup header. It would be better if you resize your popup to fix the issue.

**Your popup images:**

![image](https://user-images.githubusercontent.com/80612925/180606046-a860c951-e9a8-48f8-b960-1566373f4c42.png)
![image](https://user-images.githubusercontent.com/80612925/180606056-91fdab24-ea8e-4c80-a2b7-153d53c968db.png)